tinymce/widgets.py

```diff
@@ -100,7 +100,7 @@ def _media(self):
     media = property(_media)
 
 
-class AdminTinyMCE(admin_widgets.AdminTextareaWidget, TinyMCE):
+class AdminTinyMCE(TinyMCE, admin_widgets.AdminTextareaWidget):
```

Comment:
I wonder if ``TinyMCE`` should continue to extends ``forms.Textarea`` in that case or if we should just provide a mixin (inheriting from object)